example/spec/parallel/addUniqueGranuleId/AddUniqueGranuleIdSpec.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
```

Comment:
This is a component integration test that validates that for some set of input the component runs, interops with CMA and generates an expected payload.